blog/2022/operator-crs.adoc

```diff
@@ -0,0 +1,26 @@
+:title: The future of Keycloak Operator CRs
+:date: 2022-08-10
+:publish: true
+:author: Václav Muzikář
+
+A while back, we've https://www.keycloak.org/2021/10/keycloak-x-update.html#_operator_and_containers[announced] a new
+Operator rewritten from scratch to provide the best experience for the Quarkus distribution. While the
+https://github.com/keycloak/keycloak-operator[legacy Operator] is now deprecated and
+https://www.keycloak.org/2022/03/releases.html[will reach EOL with Keycloak 20], the
+https://github.com/keycloak/keycloak/tree/main/operator[new one] is already available as
+a preview, see the https://www.keycloak.org/operator/installation[installation guide].
+
+One of the most common concerns around the new Operator is the current lack of the CRs for managing Keycloak resources,
+such as Realm, Users and Clients, in a cloud-native way. One of the key aspects of the new Operator will be the reimagined
+way for managing these Keycloak resources via new CRs and git-ops. This new approach will leverage
+the https://www.keycloak.org/2022/07/storage-map.html[new storage architecture] and future immutability options,
+making the CRs the declarative single source of truth. In comparison to the legacy Operator, this will bring high robustness,
+reliability, and predictability to the whole solution.
+
```

Comment:
```suggestion
One of the most common concerns around the new Operator is the current lack of the CRDs for managing Keycloak resources, such as realm, users and clients, in a cloud-native way. One of the key aspects of the new Operator will be redesign
of managing these Keycloak resources via CRs and git-ops. This new approach will leverage
the https://www.keycloak.org/2022/07/storage-map.html[new storage architecture] and future immutability options,
making the CRs the declarative single source of truth. In comparison to the legacy Operator, this will bring high robustness,
reliability, and predictability to the whole solution.
```

Immutability is far more wired-in than in the storage, see comment below.